integration/scheduler.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: page
+title: Custom Scheduler Integration
+subtitle: Integrating a Custom Scheduler with the Observatory Control System
+---
+
+# Custom Scheduler Integration
+For users who wish to implement their own scheduler, this section describes that workflow.
+
+The role of the scheduler is to turn observation requests into scheduled observations for a telescope to perform. A scheduler brings together the current set of schedulable requests, site telemetry and telescope availability to generate a schedule of observations.
+
+At its most basic level, a scheduler should be able to: 
+* Retrieve all schedulable requests for a given time window
+* Take a set of observation requests and turn it into a set of observations to be executed on a telescope
+* Determine if a target is observable within the time window specified by a request, if it is, consider it for scheduling. The [OCS rise_set library](https://github.com/observatorycontrolsystem/rise_set) can be helpful in determining observability of a target from a given location.
+* Create an observation for each scheduled request with a specific site, enclosure, telescope, instrument, and start and end time
+* If a schedule exists, cancel the existing schedule, excluding any running observations, so it can be replaced with a new set of observations. (HTTP POST `/api/observations/cancel`)
+* Submit the newly-generated observations (HTTP POST `/api/observations`)
+
+## Determining the Set of Schedulable Requests
+The first step to scheduling observations is for the scheduler to retrieve the set of schedulable requests from the observation portal by querying the `/api/requestgroups/schedulable_requests` endpoint.
+
+A schedulable request is defined as a request which is not in a terminal state, is part of an active proposal with enough time allocated to accommodate the request, and has a window within the current semester.
+
+TODO: Add /api/requestgroups/schedulable_requests to autogenerated docs and link here.
```

Comment:
I've PR'd this on the observation portal, and I'll add a link to it once that's deployed and merged.

https://github.com/observatorycontrolsystem/observation-portal/pull/226